packer/ansible/roles/vault/tasks/main.yml

```diff
@@ -1,25 +1,18 @@
 # The MIT License (MIT)
-#
 # Copyright (c) 2014-2019 Avant, Sean Lingren
 
 - name: Update all packages
   package:
     name: '*'
     state: latest
 
-- name: Install EPEL
-  package:
-    name: "epel-release"
-    state: latest
-
```

Comment:
It looks like awscli is the only hard dependency here for needing EPEL, but in any case, another option is to execute this conditionally based on `ansible_distribution` like this:

```
- name: Install EPEL
  package:
    name: "epel-release"
    state: latest
  when: ansible_distribution in ["CentOS", "RedHat"]
```